### ISSUE TITLE
have you tried having good ideas instead of bad ones

### ISSUE THREAD

**Issue by Kansattica** [RupertBenWiser/Web-Environment-Integrity#84]
i mean, okay, it's okay to have bad ideas. i just try not to post them on github when i do have them

**Comment by chetchaka** [comment]
It's good to make fun of these bastards, but you gotta be more accurate. This idea they've put forward, it's a bad idea on purpose. They aren't dumb fools. They are smart people and they're being purposely evil for profit by lying to your face and telling you "Oh but you misunderstand haha, this is actually something you *do* want. It's good for you. If you think otherwise, you are delusional."

**Comment by Howdoesthiswo** [comment]
Yeah it's deliberate misdirection. The actual intent is to defeat ad blockers and attain monopoly status, but they can't ADMIT that, so they list off a bunch of other reasons that this proposal won't actually achieve as red herrings.
